app/notifications/validators.py

```diff
@@ -317,8 +317,10 @@ def validate_and_format_recipient(
         return validate_and_format_email_address(email_address=send_to)
 
 
-def check_sms_content_char_count(content_count):
-    if content_count > SMS_CHAR_COUNT_LIMIT:
+def check_sms_content_char_count(content_count, service_name):
+    if (
+        content_count + len(service_name) + 2 > SMS_CHAR_COUNT_LIMIT
+    ):  # the +2 is to account for the ': ' that is added to the service name
```

Comment:
hahaha thats for the comment I was just going to comment and ask what is the +2 for